src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -321,7 +344,17 @@ private void setup(OptionSet options, Properties properties) {
 		this.kinesisStream  = fetchOption("kinesis_stream", options, properties, null);
 		this.kinesisMd5Keys = fetchBooleanOption("kinesis_md5_keys", options, properties, false);
 
-		this.outputFile         = fetchOption("output_file", options, properties, null);
+		this.outputFile = fetchOption("output_file", options, properties, null);
+
+		this.metricsReportingType = fetchOption("metrics_type", options, properties, null);
+		this.metricsSlf4jInterval = fetchLongOption("metrics_slf4j_interval", options, properties, 60L);
+		this.metricsHTTPPort = Integer.parseInt(fetchOption("metrics_http_port", options, properties, "8080"));
+		this.metricsDatadogType = fetchOption("metrics_datadog_type", options, properties, "");
```

Comment:
maybe explicitly default to "udp" since that's how the actual code seems to behave.